setup.cfg

```diff
@@ -23,9 +23,17 @@ exclude =
     .scratch,
     _vendor,
     data
-select = E,W,F
-ignore = W504
-
+    # TODO: Remove when fixing flake8 warnings
+    src/pip/*
```

Comment:
this will disable linting in pip, you don't want that

Reply:
I wanted to only add `flake8-bugbear` fixes to the tests folder, how would I achieve that.

Reply:
you could per-file-ignores src/pip/*: B maybe?

Reply:
That worked. Added in https://github.com/pypa/pip/pull/8398/commits/59b27ad0f401938a0402ab4d408a6092d681c763